src/payloads/Attendance.ts

```diff
@@ -1,22 +1,62 @@
-import { IsOptional, IsBoolean, IsNumber, IsInstance } from 'class-validator';
+import {
+  IsOptional,
+  IsBoolean,
+  IsNumber,
+  IsInstance,
+  ValidateNested,
+  IsInt,
+} from 'class-validator';
+import { Type } from 'class-transformer';
+
 import { AppUserEventResponse } from './AppUser';
 import { EventAttendanceResponse } from './Event';
 
+export class AttendanceCheckOffRequest {
+  @IsInt()
+  readonly attendeeId: number;
```

Comment:
we need to commit to either Id or ID, i think there's both in the codebase rn

Reply:
Yeah that's true, I personally prefer Id but ID might be used in the codebase more...

Reply:
Okay so Id has 53 results in 12 files of hkn-server and ID has 99 results in 17 files of hkn-server. If we do change Id -> ID, I think we might have to be careful because the stuff from typeorm uses Id instead of ID